src/sentry/static/sentry/app/views/organizationGroupDetails/groupSimilarIssues/similarTraceID/header.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Clipboard from 'app/components/clipboard';
+import {IconCopy} from 'app/icons';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+
+type Props = {
+  traceID?: string;
+};
+
+const Header = ({traceID}: Props) => (
+  <Wrapper>
+    <h4>{t('Issues with the same trace ID')}</h4>
+    {traceID ? (
+      <Clipboard value={traceID}>
+        <ClipboardWrapper>
+          <span>{traceID.slice(0, 7)}</span>
```

Comment:
The "Results" table in Discover also displays a shorter version of the eventID ... the difference is that it was 8 instead of 7 characters. Now I created the `getShortEventId` function that is being used in both places 😉